tensorflow/compiler/xla/service/gpu/gpu_conv_runner.h

```diff
@@ -68,6 +69,7 @@ struct GpuConvConfig {
   Shape filter_shape;
   Shape output_shape;
   std::optional<FusionConfig> fusion;
+  std::string serialized_graph;
```

Comment:
Comment was this is, and state it's only used for FP8 convolutions. Maybe state that the comment in cudnn_fused_conv_rewriter.cc has more info on this.